forge-gui/res/editions/Ravnica Remastered.txt

```diff
@@ -3,6 +3,7 @@ Code=RVR
 Date=2024-01-12
 Name=Ravnica Remastered
 Type=Reprint
+Booster=10 Common:!fromSheet("RVR Mana Fixing"):fromsheet("RVR cards"), 3 Uncommon:!fromSheet("RVR Mana Fixing"):fromSheet("RVR cards"), 1 RareMythic:fromSheet("RVR cards"), 1 fromSheet("RVR Mana Fixing")
```

Comment:
Would be preferable to include the retro cards too but not a massive issue, up to you

Reply:
It's tricky to implement it properly. They change their approach on collation more or less every (special) set in at least some subtle ways.

Regarding retro cards:
- there is one guaranteed retro card in every draft booster
- if the retro card is rare or mythic, it replaces the regular rare/mythic
- if the retro card is uc/c, it replaces a common
- from my experience opening paper draft boosters, it does not affect the mana fixing slot
- the foil slot (33% of all boosters) replaces a common and can be retro or regular
- not all retro cards appear in draft boosters

Also, there is a small chance of getting a borderless anime card in the rare/mythic slot.

Reply:
Ahh the retro cards are just the frame variation of cards already in the booster so I think it's fine to exclude them. I think this PR is good to go! Thanks!